lib/chef/provider/package/cab.rb

```diff
@@ -74,7 +74,7 @@ def remove_package(name, version)
 
         def dism_command(command)
           with_os_architecture(nil) do
-            result = shell_out("dism.exe /Online /English #{command} /NoRestart", { timeout: new_resource.timeout })
+            result = shell_out("dism.exe /Online /English #{command} /NoRestart", { timeout: new_resource.timeout || 3600 })
```

Comment:
I am only concerned about timeout to be ORed 3600. During testing on 2016 I found the package to get installed in nearly 45mins. So I've added here.

Reply:
If we want to change the default we should define a timeout property in the msu_package resource and set a new default. Doing it this way hides it from our documentation generation so no user will know there's a new default.

Reply:
Thanks @tas50 for suggestion. Will be adding `timeout` property to override default.